prime-dotnet-webapi/Startup.cs

```diff
@@ -236,10 +236,13 @@ protected void ConfigureClients(IServiceCollection services)
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public void Configure(IApplicationBuilder app, IWebHostEnvironment env, IHostApplicationLifetime lifetime)
         {
-            if (env.IsDevelopment())
-            {
-                app.UseDeveloperExceptionPage();
-            }
+            // ********************************* *********************************
+            // Leaving the code block below here and will remove after PR approval
+            // ********************************* *********************************
+            // if (env.IsDevelopment())
+            // {
+            //     app.UseDeveloperExceptionPage();

```

Comment:
For my own understanding what is the worry around this in DEV?

Reply:
Hi, this me from the future!
I am not 100%, sure, I have to read more about that extension; however, that seems to capture and handle all unhandled exceptions is my surface understanding.

Reply:
That indicates that Sentry should be set up as middleware to allow it capture the exception before it's handled by the default handler.

Reply:
Found this with a bit of searching [Sentry Middleware](https://aevitas.medium.com/logging-to-sentry-in-asp-net-core-8c84c5c592db).  He provides a middleware example and places its execution before the default handler on the response.

Reply:
Yes, the plan will be to use Sentry as middleware since we now want to use it in Dev

Reply:
and I'm assuming Test is already implements.  Seems we're mentioning Dev and Prod a lot, but we want this to across all environments.

Reply:
Yes, so for now it is not limited to any environment, the ticker initially mentioned that it is for Prod but this is not longer the case anymore.